<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="aspire512" name="Fluid PUMA" tests="4" failures="3" timestamp="27 May 2016 04:23:21 GMT" time="653.465" errors="0">
  <testcase name="tcPUMADemo01" time="514.344" classname="com.puma.testscripts.Fluid_PUMA_Demo">
    <failure type="java.lang.AssertionError" message="Test Failed. Order Placed not available in Order Summary Page">
      <![CDATA[java.lang.AssertionError: Test Failed. Order Placed not available in Order Summary Page
	at org.testng.Assert.fail(Assert.java:94)
	at com.ecom.support.Log.fail(Log.java:594)
	at com.ecom.support.Log.assertThat(Log.java:777)
	at com.puma.testscripts.Fluid_PUMA_Demo.tcPUMADemo01(Fluid_PUMA_Demo.java:96)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:497)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:84)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:714)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:901)
	at org.testng.internal.TestMethodWithDataProviderMethodWorker.call(TestMethodWithDataProviderMethodWorker.java:73)
	at org.testng.internal.TestMethodWithDataProviderMethodWorker.call(TestMethodWithDataProviderMethodWorker.java:14)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
	at java.lang.Thread.run(Thread.java:745)
]]>
    </failure>
  </testcase> <!-- tcPUMADemo01 -->
  <testcase name="tcPUMADemo01" time="570.284" classname="com.puma.testscripts.Fluid_PUMA_Demo"/>
  <testcase name="tcPUMADemo01" time="502.619" classname="com.puma.testscripts.Fluid_PUMA_Demo">
    <failure type="java.lang.AssertionError" message="Test Failed. Order Placed not available in Order Summary Page">
      <![CDATA[java.lang.AssertionError: Test Failed. Order Placed not available in Order Summary Page
	at org.testng.Assert.fail(Assert.java:94)
	at com.ecom.support.Log.fail(Log.java:594)
	at com.ecom.support.Log.assertThat(Log.java:777)
	at com.puma.testscripts.Fluid_PUMA_Demo.tcPUMADemo01(Fluid_PUMA_Demo.java:96)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:497)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:84)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:714)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:901)
	at org.testng.internal.TestMethodWithDataProviderMethodWorker.call(TestMethodWithDataProviderMethodWorker.java:73)
	at org.testng.internal.TestMethodWithDataProviderMethodWorker.call(TestMethodWithDataProviderMethodWorker.java:14)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
	at java.lang.Thread.run(Thread.java:745)
]]>
    </failure>
  </testcase> <!-- tcPUMADemo01 -->
  <testcase name="tcPUMADemo01" time="606.383" classname="com.puma.testscripts.Fluid_PUMA_Demo">
    <failure type="java.lang.AssertionError" message="Test Failed. Order Placed not available in Order Summary Page">
      <![CDATA[java.lang.AssertionError: Test Failed. Order Placed not available in Order Summary Page
	at org.testng.Assert.fail(Assert.java:94)
	at com.ecom.support.Log.fail(Log.java:594)
	at com.ecom.support.Log.assertThat(Log.java:777)
	at com.puma.testscripts.Fluid_PUMA_Demo.tcPUMADemo01(Fluid_PUMA_Demo.java:96)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:497)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:84)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:714)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:901)
	at org.testng.internal.TestMethodWithDataProviderMethodWorker.call(TestMethodWithDataProviderMethodWorker.java:73)
	at org.testng.internal.TestMethodWithDataProviderMethodWorker.call(TestMethodWithDataProviderMethodWorker.java:14)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
	at java.lang.Thread.run(Thread.java:745)
]]>
    </failure>
  </testcase> <!-- tcPUMADemo01 -->
</testsuite> <!-- Fluid PUMA -->
